src/sensors/Particles.js

```diff
@@ -125,6 +125,6 @@ function finishedUpdate(particles, n)
     particles.attribs.alpha.needsUpdate = true;
 
     if(n>particles.max_pts){
-        throw 'Attempted to draw more points than max_pts allows';
+        console.warn('Attempted to draw more points than max_pts allows');
```

Comment:
I would make this `console.error` instead actually. Then the effect will be the same, but at least your script execution won't halt